game_eggs/steamcmd_servers/sonsoftheforrest/egg-sons-of-the-forest.json

```diff
@@ -0,0 +1,154 @@
+            "container": "ghcr.io\/parkervcp\/installers:debian",
+            "entrypoint": "bash"
+        }
+    },
+    "variables": [
+        {
+            "name": "QueryPort",
+            "description": "UDP port used by Steam to list the server and enable the discovery services.",
+            "env_variable": "QUERY_PORT",
+            "default_value": "27016",
+            "user_viewable": true,
+            "user_editable": false,
+            "rules": "required|integer|between:1024,65536",
+            "field_type": "text"
+        },
+        {
+            "name": "BlobSyncPort",
+            "description": "BlobSyncPort UDP port used by the BlobSync system to initialize game systems and exchange data.",
+            "env_variable": "BLOBSYNC_PORT",
+            "default_value": "9700",
+            "user_viewable": true,
+            "user_editable": false,
+            "rules": "required|integer|between:1024,65536",
+            "field_type": "text"
+        },
+        {
+            "name": "Server Name",
+            "description": "Name of the server visible in the server list, and in the Steam contacts.",
+            "env_variable": "SRV_NAME",
+            "default_value": "Pterodactyl hosted Server",
+            "user_viewable": true,
+            "user_editable": true,
+            "rules": "required|string|max:60",
+            "field_type": "text"
+        },
+        {
+            "name": "Max Players",
+            "description": "The maximum number of players allowed simultaneously on the server.",
+            "env_variable": "MAX_PLAYERS",
+            "default_value": "4",
+            "user_viewable": true,
+            "user_editable": true,
+            "rules": "required|integer|min:1",
```

Comment:
Maximum of 8 Players allowed

Reply:
ple provide a link ;)

Reply:
https://steamcommunity.com/sharedfiles/filedetails/?id=2992700419&snr=1_5_9_ here it is :)

Reply:
![image](https://github.com/parkervcp/eggs/assets/13904508/720f4ace-88a5-4746-a053-8baf7cfa1c88)